### PR TITLE
Rewrite About view

### LINKED ISSUE #449: dev: rewrite About new
We want to rewrite the About view in Settings to look modern and in a horizontal layout

Some apps with good about views we could take as reference
- Xcode (Apple)
- Pages (Apple)
- [Cork](https://github.com/buresdv/Cork)
- [CodeEdit](https://github.com/CodeEditApp/CodeEdit)
- [Proxyman](https://proxyman.io)

### PR DESCRIPTION
**Pull request by Jerry23011**
Closes #449 

## Preview
截屏2024-05-03 17 13 22
截屏2024-05-03 17 13 28

## Known Issues
- The copyright info is not localized since it's directly extracted from the Info.plist
- ~~There seem to be no resize animations while switching from other tabs~~
- ~~The about view of the main app is still yet to be implemented, I don't know how to write the objc code for AppDelegate~~

### CODE REVIEW

**Review by phlpsong — DISMISSED** [review]
LGTM

File: Easydict/Swift/View/SettingView/Tabs/TabView/AboutTab.swift [hunk 1]
```diff
@@ -6,35 +6,73 @@
...
+}
+
+// MARK: - AboutTab
+
 @available(macOS 13, *)
 struct AboutTab: View {
     // MARK: Internal
 
     var body: some View {
         ScrollView {
```
Comment (AkaShark) [inline Easydict/Swift/View/SettingView/Tabs/TabView/AboutTab.swift:28]: you can use `ScrollView` wrap to resolve `There seem to be no resize animations while switching from other tabs` like https://github.com/tisfeng/Easydict/pull/532/commits/a667fb1574f7ae38ea9d9b8119cd9cf206927760

Comment (Jerry23011) [reply Easydict/Swift/View/SettingView/Tabs/TabView/AboutTab.swift:28]: This resolved the animation issue but the about view in the menubar doesn't seem to support scrollview, or perhaps some additional handling needs to be done?
截屏2024-05-03 18 10 00

Comment (AkaShark) [reply Easydict/Swift/View/SettingView/Tabs/TabView/AboutTab.swift]: I saw your new commit https://github.com/tisfeng/Easydict/pull/532/commits/6786892f5c2d8ec2d705b348918875103b7f1a02. Seems the way can resolve this problem.

**Comment by tisfeng** [comment]
I just improved the About page.

image


image

**Comment by Jerry23011** [comment]
Yes, I built itand they LGTM

**Comment by Jerry23011** [comment]
Whenever I close the about view window from the menubar, easydict crashes. I'm not sure if it's just on my machine
截屏2024-05-03 23 27 58

**Comment by Jerry23011** [comment]
Bing gave me this code, worked on my end 🥹
```swift
aboutWindow.isReleasedWhenClosed = false
```

**Comment by tisfeng** [comment]
> Bing gave me this code, worked on my end 🥹
> 
> ```swift
> aboutWindow.isReleasedWhenClosed = false
> ```

Yes, it seems to work, I improved the code, please check it.

**Review by AkaShark — APPROVED** [review]

**Comment by Jerry23011** [comment]
This library looks very cool: [Vortex](https://github.com/twostraws/Vortex)
I'll try to implement it if we're not in a hurry for release. Can be a pretty nice 彩蛋
https://github.com/tisfeng/Easydict/assets/89069957/b2c7abde-33a1-4d70-8b04-707e04edc0fb

**Comment by tisfeng** [comment]
> > Yes, it seems to work, I improved the code, please check it.
> 
> Runs without issues on my end
> 
> This library looks very cool: [Vortex](https://github.com/twostraws/Vortex) I'll try to implement it if we're not in a hurry for release. Can be a pretty nice 彩蛋
> 
>  2024-05-04.00.42.19.mov

ok, it looks cool, you can add it later.

**Pull request merged**